apps/web/src/app/providers/DocumentProvider.tsx

```diff
@@ -24,20 +24,15 @@ const DocumentVersionProvider = ({
   commitUuid: string
 }) => {
   const { data: documents } = useDocumentVersions({
-    projectId,
-    commitUuid,
+    projectId: projectId,
+    commitUuid: commitUuid,
   })
-
-  const document = useMemo(() => {
-    return (
-      documents?.find((d) => d.id === fallbackDocument.id) ?? fallbackDocument
-    )
-  }, [documents, fallbackDocument])
+  const document = documents.find((d) => d.id === fallbackDocument.id)
```

Comment:
Why the memo removal?

Reply:
there's a fallback in the context setting